c10/util/Float8_e8m0fnu.cpp

```diff
@@ -0,0 +1,10 @@
+#include <c10/macros/Macros.h>
+#include <c10/util/Float8_e8m0fnu.h>
+
+namespace c10 {
+
+static_assert(
```

Comment:
Can we have these in a single shared cpp file built with macro to remove the need for a new cpp file?

Reply:
tracking in https://github.com/pytorch/pytorch/issues/146647